internal/utils/helper.go

```diff
@@ -58,8 +62,7 @@ func GetLatestSuccessfulRelease(releases map[string]*types.Release) *types.Relea
 
 //CheckPort takes a port number and checks if its available.
 //If available, will return the port as it is. If not, it will terminate the CLI with error.
-func CheckPort(port int) int {
-
+func CheckIfPortAvailable(port int) int {
```

Comment:
comment - why not used this one? U already have a function checking the port apparently?